### DISCUSSION TITLE
auto search, start date

### DISCUSSION THREAD

**Discussion by tommyhome** [airdcpp/airdcpp-windows#107] [Ideas]
Hi,
For auto search there is the option to specify an expiry date, I would like to suggest adding a start date.
Why you might wonder, consider the following: if you know show X premiers on a specific date then you can set it to start searching that date. That would limit the amount of unnecessary searches.

**Comment by vildsvin666** [comment]
Indeed, you got a point. Consider RSS feed?

**Comment by Abnormalpingu** [comment]
Hi, to add to this, a lot of shows have pauses in the middle of a season (for example https://next-episode.net/elsbeth/season-2), second to last episode is on the 19.dec and the last episode is the 30th of january. 
This would make it possible to define the auto search for the next search on the 30th instead of pausing the auto search and manually make sure the episode gets pulled (kind of defeats the auto in auto-search). 
Normally it would make sense that episodes are available bi-weekly which is fine, but this is just to give a more complete functionality for the auto-search. Its not a huge deal but it is a reasonable functionality improvement.
